tests/test_scene.py

```diff
@@ -53,6 +53,45 @@ def test_cosmos_basic():
     except ImportError:
         print 'The assert_raises tests require nose'
 
+    # Try making galaxies
+    gal_real = cat2.makeGalaxy(index=0,gal_type='real',chromatic=False)
+    if not isinstance(gal_real, galsim.RealGalaxy):
+        raise TypeError("COSMOS Catalog makeGalaxy routine does not return an instance of"+\
+                        "'galsim.RealGalaxy'")
```

Comment:
You don't need the `\` here.  Or the `+` for that matter.  Python automatically concatenates two consecutive strings.

However, you should put a space at the end of the first string.  Otherwise you'll get "... of'galsim.RealGalaxy'".